examples/tracetest-opensearch/README.md

```diff
@@ -0,0 +1,16 @@
+# Tracetest + Opensearch
+
+This repository objective is to show how you can configure your tracetest instance to connect to an opensearch instance and use it as its tracing backend.
+
+## Steps
+
+1. [Install the tracetest CLI](https://github.com/kubeshop/tracetest/blob/main/docs/installing.md#cli-installation)
+2. Run `tracetest configure` on a terminal and type: `http://localhost:8080` to make your CLI send all requests to that address
```

Comment:
Do we need to tell the user what the expectations are for location of OpenSearch and that they may need to look at X, Y, and Z config file to ensure it matches their environment? It looked like there were some hardcoded paths such as http://opensearch:9200 that could be different in different environments